packages/plugins/htpasswd/tests/htpasswd.test.ts

```diff
@@ -1,36 +1,35 @@
 /* eslint-disable jest/no-mocks-import */
+// @ts-ignore: Module has no default export
+import bcrypt from 'bcryptjs';
+// @ts-ignore: Module has no default export
 import crypto from 'crypto';
-// @ts-ignore
+// @ts-ignore: Module has no default export
```

Comment:
Thanks for the note, I'll double check on my #2985 migration

Reply:
Yeah, I was going to fix the actual `import` statement but then trying to override any member of these packages with a `jest.fn()` would result in:
```javascript
import * as crypto from 'crypto';
```

![image](https://user-images.githubusercontent.com/5667395/156640703-f726de7b-ea9e-4090-bbc3-b5fc32a335b5.png)

And the tests would fail as well since the members of the package could not be overridden.